addresses/mainnet.json

```diff
@@ -414,6 +414,22 @@
           "fee": "0.003",
           "poolId": "0x7173b184525fead2ffbde5fbe6fcb65ea8246ee70002000000000000000000c7"
         }
+      },
+      {
+        "address": "0xCFe60a1535ecc5B0bc628dC97111C8bb01637911",
+        "trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+        "ptPool": {
+          "address": "0x56df5ef1A0A86c2A5Dd9cC001Aa8152545BDbdeC",
+          "poolId": "0x56df5ef1a0a86c2a5dd9cc001aa8152545bdbdec000200000000000000000168",
+          "fee": "0.1",
+          "timeStretch": 88.75
+        },
+        "weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+        "convergentCurvePoolFactory": {
```

Comment:
this seems wrong, this should just be an address no?  this is probably a result of something i did earlier

Reply:
Updated to just the address. Our scripts are automatically writing to file like that, so I'll push a separate fix to the script